### PR TITLE
Improved ISO15118-2 support

### PR DESCRIPTION
**Pull request by SebaLukas**
- This PR is a requirement for PR#72 ( #https://github.com/EVerest/everest-core/pull/72)
- Added new Everest EVSE and EV controller
- Supported the Everest iso15118_charger and iso15118_ev interfaces
- Full support for ISO-2 AC charging (EIM)
- Initial work on ISO-2 DC until PreCharge

Signed-off-by: Sebastian Lukas [sebastian.lukas@pionix.de](mailto:sebastian.lukas@pionix.de)
Signed-off-by: Cornelius Claussen [cc@pionix.de](mailto:cc@pionix.de)

### CODE REVIEW

File: RISE-V2G-EVCC/EVCCConfig.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ###############################################################################
 #  The MIT License (MIT)
 #
-#  Copyright (c) 2015 - 2019  Dr. Marc M�ltin (V2G Clarity)
+#  Copyright (c) 2015 - 2019  Dr. Marc M�ltin (V2G Clarity)
```
Comment (hikinggrass) [inline RISE-V2G-EVCC/EVCCConfig.properties]: Encoding error?

File: RISE-V2G-EVCC/src/main/java/com/v2gclarity/risev2g/evcc/main/StartEVCC.java [hunk 2]
```diff
@@ -23,15 +23,40 @@
...
 public class StartEVCC {
 
 	public static void main(String[] args) {
+		final Logger logger = LogManager.getLogger(StartEVCC.class.getSimpleName());
 		MiscUtils.loadProperties(GlobalValues.EVCC_CONFIG_PROPERTIES_PATH.toString());
-		new V2GCommunicationSessionHandlerEVCC();
+
+		// *** EVerest code start ***
+		logger.info("STARTED WITH ARGS:");
+        for(String arg : args) {
```
Comment (hikinggrass) [inline RISE-V2G-EVCC/src/main/java/com/v2gclarity/risev2g/evcc/main/StartEVCC.java]: Indentation looks off

File: RISE-V2G-EVCC/src/main/java/com/v2gclarity/risev2g/evcc/transportLayer/TCPClient.java [hunk 3]
```diff
@@ -132,6 +133,9 @@ public void send(V2GTPMessage message, int timeout) {
 	@Override
 	public void stop() {
 		if (!isStopAlreadyInitiated()) {
+
+			ObjectHolder.ev_mqtt.mqtt_disconnect(); //TODO_SL: Testen ob das so funktioniert.
```
Comment (hikinggrass) [inline RISE-V2G-EVCC/src/main/java/com/v2gclarity/risev2g/evcc/transportLayer/TCPClient.java]: Does it? :grin:

File: RISE-V2G-SECC/SECCConfig.properties [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ###############################################################################
 #  The MIT License (MIT)
 #
-#  Copyright (c) 2015 - 2019  Dr. Marc M�ltin (V2G Clarity)
+#  Copyright (c) 2015 - 2019  Dr. Marc M�ltin (V2G Clarity)
```
Comment (hikinggrass) [inline RISE-V2G-SECC/SECCConfig.properties]: Encoding error

**Review by hikinggrass — CHANGES_REQUESTED** [review]

File: RISE-V2G-EVCC/EVCCConfig.properties (hunk 1 above)
Comment (corneliusclaussen) [reply RISE-V2G-EVCC/EVCCConfig.properties]: fixed

File: RISE-V2G-SECC/SECCConfig.properties (hunk 4 above)
Comment (corneliusclaussen) [reply RISE-V2G-SECC/SECCConfig.properties]: fixed

File: RISE-V2G-EVCC/src/main/java/com/v2gclarity/risev2g/evcc/main/StartEVCC.java (hunk 2 above)
Comment (corneliusclaussen) [reply RISE-V2G-EVCC/src/main/java/com/v2gclarity/risev2g/evcc/main/StartEVCC.java]: fixed

File: RISE-V2G-EVCC/src/main/java/com/v2gclarity/risev2g/evcc/transportLayer/TCPClient.java (hunk 3 above)
Comment (SebaLukas) [reply RISE-V2G-EVCC/src/main/java/com/v2gclarity/risev2g/evcc/transportLayer/TCPClient.java]: Yes :D

**Review by hikinggrass — APPROVED** [review]

**Pull request merged**